keyboards/handwired/hacked_motospeed/readme.md

```diff
@@ -0,0 +1,21 @@
+# hacked_motospeed
+
+This is a
+   - tiny (44key),
+   - budget (~50€ total expense),
+   - easy diy (little wiring & soldering needed),         
+   
+mechanical keyboard.   
+It is made by wiring two Motospeed K22 numpads to a Teensy++ 2.0, while reusing as much of the original PCB as possible.  
+(A detailed guide will be posted and linked later)
+
+
+Keyboard Maintainer: [Vladislav Opara](https://github.com/Deckweiss)  
+Hardware Supported: Motospeed K22 numpads wired to Teensy++ 2.0 according to my guide (upcoming).  
+Hardware Availability: Ebay, AliExpress  
```

Comment:
Any particular reason you don't recommend Amazon for this?

Reply:
I never use Amazon, I got all parts that I needed from Ebay and AliExpress.
I don't think listing every site the hardware is available on is useful or necessary.